Structure_oM/Results/Global Results/StoryDrift.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2024, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using BH.oM.Base;
+using BH.oM.Analytical.Results;
+using BH.oM.Quantities.Attributes;
+using System.ComponentModel;
+
+
+namespace BH.oM.Structure.Results
+{
+    [Description("Story Drift for a given Load Case or Load Combination.")]
+    public class StoryDrift : StructuralGlobalResult, IImmutable, IResultItem
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("Story Name.")]
+        public virtual string Story { get; }
```

Comment:
```suggestion
    [Description("The lateral displacement of the storey.")]
    public class StoreyDrift : StructuralGlobalResult, IImmutable, IResultItem
    {
        /***************************************************/
        /**** Properties                                ****/
        /***************************************************/

        [Description("Storey Name.")]
        public virtual string Storey { get; }
```
Can you double check whether the displacements are relative to the floor below? I have seen it defined as such.

Reply:
@peterjamesnugent,
thanks for commenting this.
The drift output from ETABS is considered to be the difference between the displacement of a floor and the one of the floor below. 
The drift Ratio is the ratio between such value and the height of the floor (to check against typical h/500 for example). 
I update the description above to be clearer about that and use Storey rather than Story ;)